yarn.lock

```diff
@@ -5932,11 +5932,16 @@ quick-lru@^4.0.1:
   resolved "https://registry.yarnpkg.com/quick-lru/-/quick-lru-4.0.1.tgz#5b8878f113a58217848c6482026c73e1ba57727f"
   integrity sha512-ARhCpm70fzdcvNQfPoy49IaanKkTlRWF2JMzqhcJbhSFRZv7nPTvZJdcY7301IPmvW+/p0RgIWnQDLJxifsQ7g==
 
-ramda@0.27.0, ramda@^0.27.0:
+ramda@0.27.0:
   version "0.27.0"
```

Comment:
Why would it keep two patch versions of ramda in the lockfile?

Reply:
Somewhere else in the dependency try depending on `ramda@0.27.0`

Reply:
Oh, yeah
```yaml
"@apielements/openapi3-parser@0.16.0":
  version "0.16.0"
  dependencies:
    content-type "^1.0.4"
    media-typer "^1.0.1"
    ramda "0.27.0"
    yaml-js "^0.2.3"
```